homeassistant/components/compensation/strings.json

```diff
@@ -0,0 +1,90 @@
+{
+  "config": {
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    },
+    "error": {
+      "incorrect_datapoints": "Datapoints needs to be provided in the right format, ex. '1.0, 0.0'.",
+      "not_enough_datapoints": "The number of datapoints needs to be more than the configured degree."
+    },
+    "step": {
+      "user": {
+        "description": "Add a compensation sensor",
```

Comment:
Can we very briefly explain this what a compensation sensor does?

Reply:
It's gonna be very technical.  Maybe:

> Create a compensation sensor that enhances the accuracy of a source sensors value.  This is done by calculating a new state using an nth degree polynomial with the source sensors value.